File: BarberApp.Frontend/src/app/Models/EvaluationSheet.ts
export interface EvaluationSheet{
  entryTime: string,                      // Horário de entrada
  departureTime: string,                  // Horário de saída
  details: string,                        // Detalhes
  practicePhysicalActivity: boolean,      // Pratica atividade física
  physicalActivity: string,               // Atividade física
  bodyProportions: {
    weight: number,                       // Peso
    height: number,                       // Altura
    bloodPressure: string,                // Pressão arterial
    imc: number,                          // IMC (Índice de Massa Corporal)
    classification: string,               // Classificação
    waist: number,                        // Cintura
    hip: number,                          // Quadril
    abdomen: number                       // Abdômen
  },
  observations: string,                   // Observações
  appraiser: string,                      // Avaliador
  dateOfAssessment: string,               // Data da avaliação
  certificateDelivered: string,           // Certificado entregue
  dateOfReturn: string                    // Data de retorno
}

export interface CustomObject{
  key: string,
  value: string,
}

File: BarberApp.Backend/BarberApp.DOMAIN/Models/EvaluationSheet.cs
using MongoDB.Bson.Serialization.Attributes;
using MongoDB.Bson;
using System.Text.Json.Serialization;

namespace BarberApp.Domain.Models
{
    public class EvaluationSheet
    {
        [BsonId]
        [BsonRepresentation(BsonType.ObjectId)]
        [JsonIgnore]
        [BsonElement("clientId")]
        public string Id { get; set; }
        [BsonElement("entryTime")]
        public string EntryTime { get; set; }
        [BsonElement("departureTime")]
        public string DepartureTime { get; set; }
        [BsonElement("details")]
        public string Details { get; set; }
        [BsonElement("practicePhysicalActivity")]
        public bool PracticePhysicalActivity { get; set; }
        [BsonElement("physicalActivity")]
        public string PhysicalActivity { get; set; }
        [BsonElement("bodyProportions")]
        public BodyProportions BodyProportions { get; set; }
        [BsonElement("observations")]
        public string Observations { get; set; }
        [BsonElement("appraiser")]
        public string Appraiser { get; set; }
        [BsonElement("dateOfAssessment")]
        public DateTime DateOfAssessment { get; set; }
        [BsonElement("certificateDelivered")]
        public DateTime CertificateDelivered { get; set; }
        [BsonElement("dateOfReturn")]
        public DateTime DateOfReturn { get; set; }
        [BsonElement("Checks")]
        public Checks Checks { get; set; }

    }
}
